### ISSUE TITLE
Build the Indexer Performance Epic

### ISSUE THREAD

**Issue by ian-algorand** [algorand/indexer#388]
## Summary
We have a bunch of work to do around indexer performance, a lot of which is dictated by purestake. We need to design out this epic.

## Scope/Requirements
- Get a distribution of purestake endpoints in order to see what's getting hit and what the relative performance of each is. 
- Also, which endpoints are the most important based on previous data (to help drive priority within this epic)

Basically, we need to gather information on what endpoints are used the most and out of those, which ones are slow

Deliverable here is building the epic in ZH, based on the above info.

## Urgency/Relative Priority
Pretty high priority for purestake and others, especially as indexer use continues to grow.
